.changeset/@graphql-inspector_cli-2349-dependencies.md

```diff
@@ -0,0 +1,21 @@
+---
+"@graphql-inspector/cli": patch
+---
+dependencies updates:
```

Comment:
These should be deleted, it's a bug in the tool that creates the automatic changesets :/ I'll fix and then you can delete those

Reply:
ok fixed, you can remove these and it should not create it again